cms/templates/js/content-visibility-editor.underscore

```diff
@@ -29,7 +39,7 @@
     <% if (hasExplicitStaffLock && !ancestorLocked) { %>
         <p class="tip tip-warning">
             <%- interpolate(
-                gettext('If you select an option other than "%(hide_label)s", after the subsection release date has passed, published units in this subsection will become available to learners unless units are explicitly hidden.'),
+                gettext('If you select an option other than "%(hide_label)s", published units in this subsection become available to learners unless they are explicitly hidden or the course has ended.'),
```

Comment:
Yeah, I can see that as a potential source of confusion. Would it be better just to drop that last clause and end with `If you select an option other than "%(hide_label)s", published units in this subsection become available to learners unless they are explicitly hidden`?

@catong